cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java

```diff
@@ -139,6 +142,41 @@ private class CoverageAsFeatureDatasetPoint extends ucar.nc2.ft.point.PointDatas
     }
   }
 
+  private class CoverageAsStationProfileCollection extends StationProfileCollectionImpl {
+    private VarGroup varGroup;
+
+    CoverageAsStationProfileCollection(VarGroup varGroup) {
+      super(varGroup.name + " AsStationFeatureCollection", varGroup.dateUnit, varGroup.zUnit);
+      this.varGroup = varGroup;
+      this.collectionFeatureType = varGroup.fType;
+    }
+
+    @Override
+    public IOIterator<PointFeatureCC> getCollectionIterator() throws IOException {
+      return null;
+    }
+
+    @Override
+    public PointFeatureCCIterator getNestedPointFeatureCollectionIterator() throws IOException {
+      return null;
+    }
+
+    @Override
+    protected StationHelper createStationHelper() throws IOException {
+      StationHelper helper = new StationHelper();
+      String name = String.format("GridPointRequestedAt[%s]", LatLonPoints.toString(latLonPoint, 3));
+      name = StringUtil2.replace(name.trim(), ' ', "_");
```

Comment:
does `.trim()` do anything here? I thought it only removed leading/trailing whitespace and it doesn't look like `name` has any

Reply:
I'm not sure if it's possible for `LatLongPoints.toString` to return trailing whitespace...

Reply:
won't the string be "GridPointRequestedAt[LatLonStuff]" ending with "]"? either way its a minor thing